Homework_Weak1_Day_4.txt

```diff
@@ -0,0 +1,201 @@
+problem 1 :
+
+var arr = [1,2,3,4,5,6,7,8,9,10]
+
+for ( var i = 0 ; i <= arr.length ; i++) {
+
+   for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 1;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 1 is " + result)
+     
+
+   }
+console.log("-----------------------------------") 
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 2;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 2 is " + result)
+      
+
+   }
+console.log("-----------------------------------") 
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 3;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 3 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 4;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 4 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 5;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 5 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 6;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 6 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 7;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 7 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 8;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 8 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 9;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 9 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
+
+for ( i = 0 ; i <= arr[i] ; i++ ) {
+
+    var result = arr[i] * 10;
+    
+
+     console.log("the Result of multiply " +arr[i]+ " with 10 is " + result)
+      
+
+   }
+console.log("-----------------------------------")
```

Comment:
Wow haha this is the most fun approach! Remember that you can you nested loops [read more here](https://stackoverflow.com/questions/36413159/understanding-nested-for-loops-in-javascript#answers) to accomplish the same thing with 10% of the lines. 

However, this is really good practice!